Rakefile

```diff
@@ -11,17 +12,20 @@ RSpec::Core::RakeTask.new(:spec) do |t|
 end
 
 task :prepare_sauce do
+  platform = RainforestRubyRuntime.platform
   sc =
-    case RUBY_PLATFORM
-    when /cygwin|mswin|mingw|bccwin|wince|emx/ then 'sc-windows.exe'
-    when /darwin/ then 'sc-osx'
+    case platform
+    when :windows then 'sc-windows.exe'
+    when :osx then 'sc-osx'
     else 'sc-linux'
     end
 
+  extension = platform == :windows ? '.exe' : ''
```

Comment:
Is it worth DRYing this up (with `Drivers::Sauce#apply_config` L25)? Could be a util method `sc_executable_name` on the main module.